pycode/memilio-surrogatemodel/memilio/surrogatemodel/ode_secir_groups/model.py

```diff
@@ -42,7 +42,7 @@ def plot_compartment_prediction_model(
     :param plot_col: string name of compartment to be plotted.
     :param max_subplots: Number of the simulation runs to be plotted and compared against. (Default value = 8)
     :param plot_compartment:  (Default value = 'InfectedSymptoms')
-
+    :returns: No return 
     """
     num_groups = 6
     num_compartments = 8
```

Comment:
same as above. but hardcoded values are not great...
However, we can also transfer this and the above comment to the next Pull request where we adress the secir groups directory.